packages/flutter/test/material/autocomplete_test.dart

```diff
@@ -44,7 +59,8 @@ void main() {
     User(name: 'Charlie', email: 'charlie123@gmail.com'),
   ];
 
-  testWidgets('can filter and select a list of string options', (WidgetTester tester) async {
+  testWidgets('can filter and select a list of string options',
```

Comment:
Please revert these unrelated changed lines of code, here and below.

Reply:
@shihaohong  sorry I couldn't understand this change

Reply:
Basically, this change is unrelated to the PR you're making, so please revert it to the original formatting

Reply:
thanks for clarification